### ISSUE TITLE
People: VoiceOver hints should be more specific and custom message field is inaccessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12895]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > People.
1. Swipe right to navigate through each UI element on the Users and Followers screens.
1. Try changing the role for a user other than the blog owner.
1. Try adding a follower using the Add button at top right.

**Result:** [2m7s](https://cloudup.com/ctN1uHIUEaX)

* It would be helpful to preface each part of the user data with the field name, for example "First and Last Name: Joe Smith", "Username: user1234", "Role: Administrator".
* The "+" button at top right could say, "Add a user or follower. Button."
* The custom message field on the Add a Person screen is completely inaccessible.

Note: the Invite button is read out at the top of the form and not the bottom. This is strange when navigating without sight. @iamthomasbishop what do you think about the design for how form submit buttons on pages like this work—is there anyway to keep the current visual but make the form submit button get read aloud last when using VoiceOver? Example: see "Add a Person" screenshot below.

<img src="https://user-images.githubusercontent.com/1119271/68342825-a7ec2c00-00a8-11ea-8429-02629234d3e4.png" width="200" alt="users-screen"> <img src="https://user-images.githubusercontent.com/1119271/68342830-aa4e8600-00a8-11ea-828e-7f2a02ecbc69.png" width="200" alt="followers-screen"> <img src="https://user-images.githubusercontent.com/1119271/68342837-ad497680-00a8-11ea-9319-8a914d892d8c.png" width="200" alt="invite-button-read-first"> <img src="https://user-images.githubusercontent.com/1119271/68342841-afabd080-00a8-11ea-9ff1-9888dabbe3a4.png" width="200" alt="custom-message-field"> <img src="https://user-images.githubusercontent.com/1119271/68342851-b20e2a80-00a8-11ea-8641-c3339666ecd7.png" width="200" alt="user-edit-screen">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by iamthomasbishop** [comment]
> what do you think about the design for how form submit buttons on pages like this work—is there anyway to keep the current visual but make the form submit button get read aloud last when using VoiceOver? Example: see "Add a Person" screenshot below.

I'm not sure on the technical side, but this is a very common pattern so I imagine it's a pretty straight-forward solution. @etoledom, do you have a suggestion for how we should be addressing this type of thing?

**Comment by etoledom** [comment]
I think that, if it's a known pattern in the system, then the user will be familiar with it. I can find this pattern in the Contacts app, also in Mails (even though the context is a bit different).

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
